seml/main.py

```diff
@@ -474,3 +501,6 @@ def main():
     if 'filter_states' in args:
         args.filter_states = [state.upper() for state in args.filter_states]
     f(**args.__dict__)
+
+if __name__ == "__main__":
+    main()
```

Comment:
New line missing. (Set your editor to do this stuff automatically?)

Reply:
fixed in 9d7f9f2